### DISCUSSION TITLE
Error: unknown command "zsh" for "oh-my-posh"

### DISCUSSION THREAD

**Discussion by heymands** [JanDeDobbeleer/oh-my-posh#5818] [Troubleshoot] [answered]
After the update 24.0.5, the oh-my-posh is not working. Before the update, oh-my-posh works perfectly. The zsh is configured in my .zshrc file.

OS: Arch Linux
Kernel: Linux 6.11.6

![Screenshot From 2024-11-05 10-16-09](https://github.com/user-attachments/assets/bcb461bc-8421-476c-8076-319a5cec5cb2)

![Screenshot From 2024-11-05 10-17-35](https://github.com/user-attachments/assets/0454402f-4c3d-47e0-82b6-02e84428c3dd)

**Comment by JanDeDobbeleer** [comment, marked as answer]
@heymands this syntax is now removed after being deprecated for 2 years, you can use `eval $(oh-my-posh init zsh --config xxx)"` only as of `v24.0.0`.

**Reply by heymands** [reply]
Thanks, this resolved my problem.  :) 
![Screenshot From 2024-11-05 12-13-38](https://github.com/user-attachments/assets/2688413a-b98f-4652-918f-5c40c2b3be04)
